api/tests/integration/infrastructure/repositories/stage-repository_test.js

```diff
@@ -0,0 +1,29 @@
+const { expect, databaseBuilder } = require('../../../test-helper');
+const stageRepository = require('../../../../lib/infrastructure/repositories/stage-repository');
+
+describe('Integration | Repository | StageRepository', () => {
+
+  describe('#getByCampaignId', () => {
+    it('should retrieve stage given campaignId', async () => {
+      // given
+      const tagetProfile = databaseBuilder.factory.buildTargetProfile();
+      const campaign = databaseBuilder.factory.buildCampaign();
+
+      databaseBuilder.factory.buildStage({ targetProfileId: campaign.targetProfileId, threshold : 24 });
+      databaseBuilder.factory.buildStage({ targetProfileId: campaign.targetProfileId, threshold : 55 });
```

Comment:
```suggestion
        databaseBuilder.factory.buildStage({ targetProfileId: campaign.targetProfileId, threshold : 55 });
        databaseBuilder.factory.buildStage({ targetProfileId: campaign.targetProfileId, threshold : 24 });
```
Pour mieux vérifier l'ordre de sortie

Reply:
Done 👍